grassroot-core/src/main/java/za/org/grassroot/core/util/DateTimeUtil.java

```diff
@@ -158,6 +158,20 @@ public static LocalDateTime tryParseString(String userResponse) {
 
     }
 
+    public static String checkDateString(String userInput,String mostFreqTime){
```

Comment:
Does this method have a point? If it was used earlier, and is now unused, remove it